pages/faq/troubleshooting/faq.md

```diff
@@ -138,6 +139,10 @@ Discontinued devices will no longer be actively supported by {{ $names.company.l
 
 There are a few options for devices that do not have an official device type on {{ $names.company.lower }}. If your device has an x86 architecture, you can try the [Intel NUC][nuc] image, which is built to support generic x86 devices. For other devices, you can [build your own][build-your-own] version of {{ $names.os.lower }} using our [open source repos][balenaOS]. To discuss custom board support, please contact sales@{{ $names.email_domain }}.
 
+##### What to keep in mind when choosing power supply units?
+
+Power supply units (PSUs) are a critical component to any widespread production deployment as mentioned in our [Going to Production][go-to-production] guide. When buying a PSU for your hardware, be sure to carefully inspect and understand the power requirements of the board. Current requirements (measured in Amps) are just as important as the voltage requirements (measured in Volts). A PSU that provides a voltage that is higher than required may damage the board, but a supplied voltage that is too low may cause the board to malfunction or fail to boot. If the PSU isn't capable of providing the maximum current that the board demands, then the board might eventually fail to perform under CPU-intensive loads. As the load increases, more current is drawn than the PSU is capable of supplying, which could also lead to brown-outs (i.e. the board unexpectedly resetting causing filesystem corruption). Proper research when buying PSUs or using the official adapters is recommended to prevent sceanrios like these in a production deployment.
```

Comment:
sceanrios -> scenarios

Reply:
I dunno if "official adapters" is a bit ambiguous, perhaps "official power adaptors" would be clearer?

Reply:
Improved a bunch of sentences by putting them through a checker. Resolved.